tests/splines/batched_spline_builder.cpp

```diff
@@ -68,6 +101,12 @@ using IDim = std::conditional_t<
         typename GrevillePoints<BSplines<X>>::interpolation_mesh_type,
         ddc::NonUniformPointSampling<X>>;
 #endif
+
+#if defined(BC_HERMITE)
+template <typename DerivI>
+using IDimDeriv = ddc::UniformPointSampling<DerivI>;
```

Comment:
I would rather use a simple `class DDimDeriv;`. I do not think we need a continuous dimension & discretization for the derivatives. I think only a discrete dimension is enough.

Reply:
Yes that's right! we don't need nth-order derivative with n non-integer